src/Button/Button.tsx

```diff
@@ -119,11 +122,14 @@ export class Button extends React.PureComponent<Props> {
       underlined ? "Button--linkUnderlined" : "",
     );
 
+    const aria = ariaLabel || (typeof value === "string" ? (value as string) : null);
```

Comment:
using `||` here instead of `??` because it seems `??` is not supported by all browsers